joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstCreator.scala

```diff
@@ -27,7 +32,8 @@ class AstCreator(val relativeFileName: String, val parserResult: ParserResult, v
 
   protected val logger: Logger = LoggerFactory.getLogger(getClass)
 
-  protected val scope: CSharpScope = new CSharpScope(typeMap)
+  protected val scope: CSharpScope                                   = new CSharpScope(typeMap)
+  protected val aliasToNameSpaceMapping: mutable.Map[String, String] = mutable.Map.empty
```

Comment:
I would rather modify the `NamespaceScope` class instead of introduce another variable here